frontend/src/routes/Video.tsx

```diff
@@ -343,26 +354,27 @@ const Metadata: React.FC<MetadataProps> = ({ id, event }) => {
                 <ShareButton event={event} />
             </div>
         </div>
-        <hr />
         <div css={{
             display: "grid",
             gridTemplate: "1fr / 1fr fit-content(30%)",
-            columnGap: 48,
-            rowGap: 24,
+            gap: 16,
             [`@media (max-width: ${BREAKPOINT_MEDIUM}px)`]: {
                 gridTemplate: "auto auto / 1fr",
             },
+            "> div": {
+                backgroundColor: "var(--grey95)",
+                padding: 8,
+                borderRadius: 8,
+            },
         }}>
-            <div css={{ maxWidth: 700 }}>
+            <div>
                 <Creators creators={event.creators} css={{ fontWeight: "bold" }} />
                 <Description
                     text={event.description}
                     css={{ color: "var(--grey20)", fontSize: 14 }}
                 />
```

Comment:
This was like that before but: we should restrict the height of the description box if the description gets too long. Especially on small screens, that pushes the metadata table very far down. 

I am not yet sure how to best do it. The typical YouTube solution is to make it fixed height and expand on click. For one, I would prefer it if not the whole description area acts like that "expand" button then. Rather have a dedicated clickable area (button or whatever) for it. But also, could we just make it fixed height and scrollable? One problem with nested scrollable containers: on phone you cant easily decide whether u want to scroll in the inner or outer. Maybe? One might need to test that.

Reply:
Note that this is still missing in my recent push, but I'm trying to figure something out that works well for all screen sizes and text lengths.

Reply:
Ok so for now this uses scrollable containers, but I also have a partially completed version with `show more` buttons which I might push on a different branch for comparison once that is ready.